doc/src/examples/examples_utils/create_svg_closest_points.hpp

```diff
@@ -0,0 +1,66 @@
+// Boost.Geometry
+// QuickBook Example
+
+// Copyright (c) 2021, Oracle and/or its affiliates
+// Contributed and/or modified by Vissarion Fysikopoulos, on behalf of Oracle
+
+// Use, modification and distribution is subject to the Boost Software License,
+// Version 1.0. (See accompanying file LICENSE_1_0.txt or copy at
+// http://www.boost.org/LICENSE_1_0.txt)
+
+// Code to create SVG for closest_points examples
+
+#ifndef CREATE_SVG_CLOSEST_POINTS_HPP
+#define CREATE_SVG_CLOSEST_POINTS_HPP
+
+#include <fstream>
+#if defined(HAVE_SVG)
```

Comment:
Very minor, using this define can go here :-)

Reply:
sorry, I don't get it

Reply:
Probably that you can remove the `#ifdef`. Note also that since below `boost::geometry::svg_mapper` is created technically this `#include` is needed anyway. Unless you put the whole code behind the `#ifdef`. But AFAIU is some helper utility so there is no need for that.